test/browser/integration/caniuse.js

```diff
@@ -150,29 +150,59 @@ window.caniusecb = function(caniuse) {
         o.result = o.result.valueOf();
       }
 
-      // webgl `partial` support means that not all users with these browsers
-      // have WebGL access, so we just ignore this test, and only check if the browser
+      // webgl `partial` support means that not all users with these browsers have
+      // WebGL access, so we just ignore this test, and only check if the browser
       // either fully supports or does not support
       if (o.feature === 'webgl' && o.caniuseResult.indexOf('a') === 0) {
         return;
       }
 
       // change the *documented* false positives
-      if (!ciubool && (o.feature == 'textshadow' && o.browser == 'firefox' && o.version == 3)) {
+      if (!ciubool && (o.feature == 'textshadow' && o.browser == 'Firefox' && o.version == 3)) {
         ciubool = o.fp = true;
       }
 
+      // firefox only supports web animation when a flag is enabled, which we
+      // don't do on sauce
+      if (o.feature === 'webanimations' && o.caniuseResult.indexOf('a d') === 0) {
```

Comment:
Just so I understand, whats the indexOf check doing for these single characters?